util/cmake/SeqAnSimdUtility.cmake

```diff
@@ -266,7 +267,12 @@ int main() {
   auto z = x + y;
 
   // gcc 4.9 bug
-  constexpr auto length = LENGTH<int32x8_t>::VALUE;
+  constexpr auto length1 = LENGTH<int32x4_t>::VALUE;
+  constexpr auto length2 = LENGTH<int32x8_t>::VALUE;
+  static_assert(length1 == 4u, \"\");
+  static_assert(length2 == 8u, \"\");
+  std::cout << \"length1: \" << length1 << std::endl;
+  std::cout << \"length2: \" << length2 << std::endl;
```

Comment:
How is this related to the above?

Reply:
This is the actual cmake code which should have disabled simd on gcc4.9.

But, due to compiler optimization, it did not have an effect.

Reply:
Hm, I thought (de)activation of SIMD happens via detection in the library? So we have, both, detection in library and in cmake? This is bound to be error-prone...